### PR TITLE
Shulk adjustments

### PR DESCRIPTION
**Pull request by Riah321**
Nerfs, a few buffs and some tweaks. Things looked at includes his kill power, cheese, frame data, and attributes. Some much needed love (and nerfs) were also given to his Monado Arts.

Shulk players will now be properly rewarded for learning their tech and utilizing their core gimmick. He will also have to play neutral like a real character!

Minor hitbox cleanup and repositioning. Accurate and thorough changelog below. Eldritch horror not included! :) 

Special thanks to Hypothesis, CrossSockDance and Askew for helping me make this while I wait for my switch to arrive in the mail.

[Shulk.Adjustments.txt](https://github.com/HDR-Development/HewDraw-Remix/files/9542010/Shulk.Adjustments.txt)

### CODE REVIEW

**Comment by IRBDaUrges** [comment]
Love these Changes :)

**Comment by Riah321** [comment]
~~Shulk is officially finished! Changelog is still up to date.~~
_Mistaaaaakes_

File: fighters/shulk/src/acmd/ground.rs [hunk 1]
```diff
@@ -11,19 +11,17 @@ unsafe fn shulk_attack_dash_game(fighter: &mut L2CAgentBase) {
...
+        ATTACK(fighter, 4, 0, Hash40::new("haver"), 12.0, 70, 100, 0, 35, 4.0, 0.0, 6.0, 0.0, None, None, None, 1.0, 1.0, *ATTACK_SETOFF_KIND_ON, *ATTACK_LR_CHECK_POS, false, 0, 0.0, 0, false, false, false, false, true, *COLLISION_SITUATION_MASK_GA, *COLLISION_CATEGORY_MASK_ALL, *COLLISION_PART_MASK_ALL, false, Hash40::new("collision_attr_cutup"), *ATTACK_SOUND_LEVEL_M, *COLLISION_SOUND_ATTR_CUTUP, *ATTACK_REGION_SWORD);
+        ATTACK(fighter, 5, 0, Hash40::new("haver"), 12.0, 70, 100, 0, 35, 3.5, 0.0, 14.5, 0.0, Some(0.0), Some(10.0), Some(0.0), 1.0, 1.0, *ATTACK_SETOFF_KIND_ON, *ATTACK_LR_CHECK_POS, false, 0, 0.0, 0, false, false, false, false, true, *COLLISION_SITUATION_MASK_GA, *COLLISION_CATEGORY_MASK_ALL, *COLLISION_PART_MASK_ALL, false, Hash40::new("collision_attr_cutup"), *ATTACK_SOUND_LEVEL_M, *COLLISION_SOUND_ATTR_CUTUP, *ATTACK_REGION_SWORD);
     }
-    wait(lua_state, 2.0);
+    wait(lua_state, 1.0);
```
Comment (WuBoytH) [inline fighters/shulk/src/acmd/ground.rs]: You got rid of hitboxes 1 and 2, but didn't remove the AttackModule::clear calls for those hitboxes.

**Review by WuBoytH — CHANGES_REQUESTED** [review]
Looks good, but I left one comment on something that should be changed for readability.

File: fighters/shulk/src/acmd/ground.rs (hunk 1 above)
Comment (Riah321) [reply fighters/shulk/src/acmd/ground.rs]: o7

**Review by WuBoytH — DISMISSED** [review]

**Review by SkewedAskew — DISMISSED** [review]
Remove the Toon Link files and param changes, otherwise lgtm

**Comment by SkewedAskew** [comment]
Changelog: [Shulk_changes.txt](https://github.com/HDR-Development/HewDraw-Remix/files/9793981/Shulk_changes.txt)

Hitbox Comparisons: https://imgur.com/a/1eN1p1f (does not contain a couple minor last-minute changes)

**Comment by hdr-balance-backroom** [comment]
APPROVED

**Comment by hdr-balance-backroom** [comment]
APPROVED

**Comment by hdr-balance-backroom** [comment]
APPROVED

**Comment by hdr-balance-backroom** [comment]
APPROVED

**Comment by hdr-balance-backroom** [comment]
APPROVED

**Comment by hdr-balance-backroom** [comment]
APPROVED

**Comment by hdr-balance-backroom** [comment]
APPROVED

**Comment by hdr-balance-backroom** [comment]
APPROVED

**Review by hdr-balance-backroom — APPROVED** [review]

**Review by techyCoder81 — APPROVED** [review]
lgtm

**Pull request merged**